Gemfile

```diff
@@ -13,6 +13,7 @@ gem "lita-pull-requests", github: "platanus/lita-pull-requests"
 gem "lita-slack-actions", github: "platanus/lita-slack-actions"
 gem "lita-trainer-bot", github: "platanus/lita-trainer-bot"
 gem "lita_time_tracker", github: "platanus/lita_time_tracker"
+gem "lita-harvest-tracker", github: "platanus/lita-harvest-tracker"
```

Comment:
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem lita-harvest-tracker should appear before lita_time_tracker.